txpool/src/config.rs

```diff
@@ -0,0 +1,16 @@
+#[derive(Debug, Clone)]
+pub struct Config {
+    /// Maximum number of transaction inside pool
+    pub max_tx: usize,
+    /// max depth of connected UTXO excluding contracts
+    pub max_depth: usize,
```

Comment:
should we also have a max size in terms of bytes? That way nodes don't get DOS'd by tx with all their fields filled out to the max?

Reply:
Totaly agree, added it in `future work` readme section.